client/state/notices/middleware.js

```diff
@@ -143,6 +144,15 @@ export const onPublicizeConnectionUpdateFailure = ( dispatch, { error } ) => dis
 	} ) )
 );
 
+// TODO:
+// Ideally, this error should not be shown if the live chat support is configured properly.
+// However, at the moment the olark-related data is not fully in the redux store, so there
+// is no clean way to access it in this middleware. Once it is in, we should rewrite this function
+// to take that into account.
+const onTicketSupportConfigurationRequestFailure = dispatchError(
+	translate( 'There is an error while configuring the email support form.' )
```

Comment:
This seems like it could be a weird/confusing experience for users who are eligible for live chat, and see this notice. Since it doesn't happen in response to a user action, it might be better handled somewhere other than a global notice anyway. Maybe a notice component at the top of the contact form? That form also has access to the old olark store and could use it to decide whether to display the notice.